### PR TITLE
Reset Password flow

### PR DESCRIPTION
**Pull request by xuhanwen97**
Implemeneted Reset password flow

### CODE REVIEW

File: src/views/Auth/ResetPassword.jsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                          <Label for="exampleEmail" style={{color: "white"}}>Email</Label>
+                          <Input type="email"
+                                 name="email"
+                                 id="email"
+                                 placeholder="email"
+                                 style={{color: "white"}}
+                                 value={this.state.email}
+                                 onChange={(e) => this.handleEmailChange(e)}/>
+                        </FormGroup>
+                        {this.state.passwordsEmpty && <p style={{color: "white"}}>Passwords are empty</p>}
```
Comment (dbecker1) [inline src/views/Auth/ResetPassword.jsx]: We should use reactstrap alerts for this.

File: src/views/Auth/ResetRequest.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+    var client = new APIClient();
+    client.requestResetPassword(this.state.email).then(() =>{
+      console.log("reset password request succeeded")
+    }).catch(() => {
+      console.log("reset password request failed")
+    }).finally( () => {
+      // No matter what, always go to request received page. Doesn't tell people what is a valid email!
+      this.setState({
+        requestSent: true
```
Comment (dbecker1) [inline src/views/Auth/ResetRequest.jsx]: should also set the email to "" again.

File: src/views/Auth/ResetPassword.jsx (hunk 1 above)
Comment (xuhanwen97) [reply src/views/Auth/ResetPassword.jsx]: Agreed

File: src/views/Auth/ResetRequest.jsx (hunk 2 above)
Comment (xuhanwen97) [reply src/views/Auth/ResetRequest.jsx]: agreed!

**Pull request merged**